tests/gpu.config

```diff
@@ -0,0 +1,42 @@
+/*
+========================================================================================
+    Nextflow config file for running tests
+========================================================================================
+*/
+
+params {
+    // Base directory for nf-core/modules test data
+    modules_testdata_base_path   = 's3://ngi-igenomes/testdata/nf-core/modules/'
+    // Base directory for nf-core/methylseq test data
+    pipelines_testdata_base_path = 'https://github.com/nf-core/test-datasets/raw/methylseq/'
+
+    // Input data
+    input       = "${projectDir}/assets/samplesheet_gpu.csv"
+    fasta       = "${params.pipelines_testdata_base_path}/reference/genome.fa"
+    fasta_index = "${params.pipelines_testdata_base_path}/reference/genome.fa.fai"
+    outdir      = 'results'
```

Comment:
`use_gpu = true`?

Without that, this is just a non-gpu test.

Reply:
the use_gpu param is set in the when.param block